lib/types.js

```diff
@@ -26,12 +26,7 @@ function mergeExtAttrs(a = [], b = []) {
 }
 
 // Types of types that generate an output file.
-const resolvedTypes = new Set([
-  "callback interface",
-  "dictionary",
-  "enumeration",
-  "interface"
-]);
+const resolvedTypes = new Set(["callback interface", "dictionary", "enumeration", "interface"]);
```

Comment:
I made this multi‑line so that when there’s multiple in‑flight PRs that add new types (like #123), they avoid conflicting with each other.

Reply:
I don't think that's a sufficient motivation for making the code more verbose in that way. In general, please do not change project style without opening an issue for discussion.